StashSearch/Patches/InventoryScreenShowPatch.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Aki.Reflection.Patching;
+using EFT.UI;
+using HarmonyLib;
+using System.Reflection;
+
+namespace StashSearch.Patches
+{
+    internal class InventoryScreenShowPatch : ModulePatch
```

Comment:
This felt symetric with the trader screen and was able to eliminate `Plugin.Update()`